src/components/Auth/styles.js

```diff
@@ -0,0 +1,150 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const StyledInput = styled.input`
+  width: 348px;
+  height: 48px;
+  border: none;
+  flex-grow: 0;
+  padding-left: 16px;
+  margin-top: 16px;
+  border: solid 1px #c4c4c4;
+  position: relative;
+  &:focus {
+    border: solid 1px #222;
+  }
+  ::-webkit-input-placeholder {
+    font-size: 14px;
+    line-height: normal;
+    letter-spacing: normal;
+    text-align: left;
+    color: #999;
+    height: 21px;
+    padding: 0px;
+  }
+  & + & {
+    margin-top: 1rem;
+  }
+`;
+
+export const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: black;
+  &:focus {
+    text-decoration: none;
+  }
+  &:hover {
+    text-decoration: none;
+  }
+  &:visited {
+    color: black;
+    text-decoration: none;
+  }
+  &:link {
+    color: black;
+    text-decoration: none;
+  }
+  &:active {
+    color: black;
+    text-decoration: none;
+  }
```

Comment:
요렇게 줄일 수 있어 보이는데, 아닌가요?

```suggestion
  &:visited, &:link, &:active {
    color: black;
    text-decoration: none;
  }
```

Reply:
저도 저렇게 하나로 몰아넣고 싶었는데 저장만하면 자동정렬기능때문인지 계속 이렇게 변경되서...
```
export const StyledLink = styled(Link)`
  text-decoration: none;
  color: black;
  &:focus,
  &:hover,
  &:visited,
  &:link,
  &:active {
    color: black;
    text-decoration: none;
  }
`;
```
대충 방치하려고했는데 깔끔(?) 하지않을까 해서 위에 처럼 작성 해둔겁니다. 다른 방법 있으면 찾아보겠습니다

Reply:
좋습니다~ 

아마 `prettier` 때문에 자동 정렬 되는것 같은데 `cmd + shift + 'P'` 누르시고 `Save Without Formatting` 하시면 정렬없이 저장됩니다 :)

<img width="867" alt="스크린샷 2021-12-06 오후 9 06 51" src="https://user-images.githubusercontent.com/19240202/144843267-24a79322-9ffd-4bba-9dd8-fb85b733a741.png">
`

Reply:
eeab599

Reply:
앜 수정하라는 말이 아니라 그냥 저런 방법이 있다는거 말해주고 싶었는데 ㅜㅜ 어쨌든 수고했습니다 ㅋㅋㅋ